src/game/server/ddracechat.cpp

```diff
@@ -589,6 +589,74 @@ void CGameContext::ConTimeout(IConsole::IResult *pResult, void *pUserData)
 	str_copy(pPlayer->m_TimeoutCode, pResult->GetString(0), sizeof(pPlayer->m_TimeoutCode));
 }
 
+void CGameContext::ConPractice(IConsole::IResult *pResult, void *pUserData)
+{
+	CGameContext *pSelf = (CGameContext *) pUserData;
+	if (!CheckClientID(pResult->m_ClientID))
```

Comment:
```suggestion
	if(!CheckClientID(pResult->m_ClientID))
```
There are a couple more instances of this but I'll point it out once.

Reply:
Done